### ISSUE TITLE
Haplotypes missing for run_sim_pe_reads.sh

### ISSUE THREAD

**Issue by jdidion** [bmvdgeijn/WASP#58]
The file $WASP/example_data/genotypes/chr22.hg19.haplotype.txt.gz is referenced, which does not exist in the repository, and there are no instructions on how to generate this file.

**Comment by gmcvicker** [comment]
The only place I see this file referenced is in the script:
`mapping/sim_reads/run_sim_pe_reads.sh`

which calls this script:
`mapping/sim_reads/sim_pe_reads.py`

I wrote the sim_pe_reads.py script for simulating paired end reads from a single individual's haplotypes, but it is not part of the WASP pipeline and I have not used it in some time. I left the script in the repo because I thought that it could be potentially useful or used as part of a more thorough test-suite in the future.

In any case, the haplotype file is a whitespace-delimited text file with 5 columns:
`position RefAllele AltAllele hap1 hap2`

For example the file could contain a line like:
`16050984 C G 1 0`
This would indicate that there is a SNP at position 16,050,984 of the chromosome (first base is 1, and the name of the chromosome [e.g. chr22] must be in the name of the file). The first haplotype would have the non-reference (1) allele 'G' and the second haplotype would have the reference (0) allele 'C'.

Hope this helps,
Graham

**Comment by jdidion** [comment]
I was hoping to be able to run the example from end-to-end. The example_mapping_workflow.sh references sim_pe_reads1.fq.gz and sim_pe_reads2.fq.gz, which presumably were generated by sim_pe_reads.py. Where those fastqs originally generated using chr22 SNPs from one of the YRI samples?

**Comment by gmcvicker** [comment]
I see. Yes I generated the haplotype file from a YRI sample and used that to make the sim_pe_reads, but I'm not sure which YRI sample I used and I don't have the files anymore. I agree it would be better if the example workflow was a bit more end-to-end. I think ideally it would start by downloading samples from GEO or website. Cleaning up the example data is on my todo list, but I don't have time to do this at the moment.
